src/main/java/com/groupseven/pdfproject/MainCanvas.java

```diff
@@ -77,7 +77,7 @@ public void clearRedo() {
     public void clearScreen() {
         Canvas canvas = getCanvas();
         GraphicsContext gc = canvas.getGraphicsContext2D();
-        gc.clearRect(0,0,canvas.getWidth(), canvas.getWidth());
+        gc.clearRect(0,0,canvas.getWidth(), canvas.getHeight());
```

Comment:
There was also the problem of clearing the canvas, the bottom bit was not getting cleared because of this call to `getWidth` instead of `getHeight`